Resources/Private/Fusion/Content/Carousel.fusion

```diff
@@ -0,0 +1,47 @@
+##
+# "Carousel" element
+#
+prototype(Neos.Demo:Content.Carousel) < prototype(Neos.Neos:ContentComponent) {
+
+    identifier = ${q(node).property('_identifier')}
+
+    carouselItems = Neos.Neos:ContentCollection {
+        nodePath = 'carouselItems'
+        attributes.class = 'carousel-inner'
+
+        // Each child item is augmented "item" and "active" class
+        prototype(Neos.Neos:ContentCase) {
+            @process.addItemAndActiveClass = Neos.Fusion:Augmenter {
+                class = ${'item' + (iterator.isFirst ? ' active' : '')}
+            }
+        }
+    }
+
+    carouselItemNumber = ${q(node).children('carouselItems').children().count()}
```

Comment:
I get an error here as `carouselItems` seems not to be a valid identifier anymore.

Reply:
@Sebobo, i cannot reproduce the problems you have with the carousel. Did you import the site again since i altered the nodeType of the 'carouselItems' in the export today.

Reply:
It only works when I change `carouselItems` to `carouselitems` in the mentioned line.

Reply:
I just discovered that the cr apparently stores nodeNames in lowercase so while the childNode is called 'carouselItems' for ages it was always stored as 'carouselitems'. I adjusted the nodetype and the fusion. 

@sebobo Can you test again. 

Maybe we should investigate this further since this might aswell be a CR issue. This was apparently broken for ages. Not sure how this popped up now. But the nodeName was not changed in this pr. Not sure how this ever worked.

Reply:
Turns out uppercase characters in nodeNames were never allowed and this only worked in the past because the names were passed trough renderValidNodeName. Maybe we should make the schema for nodeTypes more strict and validate that childNodes get an actual valid nodeName. (This would be a different pr for another repo offcourse).

I am quite sure now that using the lowercase `carouselitems` now in nodeType and in fusion as it is stored is the correct solution here.